run.py

```diff
@@ -2,13 +2,15 @@
 import os
 import sys
 
+def import_data():
+    from DataImport.gb_parse import parse_genbank_and_insert
+    # from DataImport.mongo_import import mongo_import_list
 
-from DataImport.gb_parse import parse_genbank
+    # mongo_import_list(parse_genbank(user_settings.INPUT), 'test_collection')
```

Comment:
No, it's what I used before - I commented it out to try the new thing and forgot to remove it.